config/initializers/okcomputer.rb

```diff
@@ -6,7 +6,7 @@
 
 OkComputer::Registry.register 'confirmation_mailer', OkComputer::ActionMailerCheck.new(ConfirmationMailer)
 OkComputer::Registry.register 'searchworks_api', OkComputer::HttpCheck.new("#{Settings.searchworks_api}/status")
-
+OkComputer::Registry.register 'background_jobs', OkComputer::SidekiqLatencyCheck.new('default', 25)
```

Comment:
Would this potentially give us false alerts if we're in a lull in requests?  (not sure what their def. of latency is here).

Reply:
Nope; from the docs:

> If you throw a lot of jobs into the queue, you can get false positives when monitoring the queue backlog. Instead, monitor the queue latency. Queue latency is the difference between when the oldest job was pushed onto the queue versus the current time. This code will check that jobs don't spend more than 30 seconds enqueued. 

https://github.com/mperham/sidekiq/wiki/Monitoring#monitoring-queue-latency

--

This is also the same check we're using on exhibits (and maybe other places?)

Reply:
👍